activesupport/lib/active_support/core_ext/securerandom.rb

```diff
@@ -36,10 +36,7 @@ def self.base58(n = 16)
   #   p SecureRandom.base36 # => "4kugl2pdqmscqtje"
   #   p SecureRandom.base36(24) # => "77tmhrhjfvfdwodq8w7ev2m7"
   def self.base36(n = 16)
-    SecureRandom.random_bytes(n).unpack("C*").map do |byte|
-      idx = byte % 64
-      idx = SecureRandom.random_number(36) if idx >= 36
-      BASE36_ALPHABET[idx]
-    end.join
+    n = n ? n.to_int : 16
```

Comment:
This line is copied from `Random::Formatter`:
https://github.com/ruby/ruby/blob/caf48487cafb8414c60f32b902decf0689ae2093/lib/random/formatter.rb#L73C5-L73C26